plugins/techdocs-backend/src/service/router.ts

```diff
@@ -226,13 +231,35 @@ export async function createRouter(
     );
   });
 
+  // Route treated similarly to /static/docs
+  // Requires an existing entity in the catalog for authorization
+  router.use(
+    '/static/entity/docs/:namespace/:kind/:name',
```

Comment:
Another option that could emphasize the separation between catalog docs and other docs: `/static/entity-docs` and `/static/custom-docs`

Reply:
see https://github.com/backstage/backstage/pull/8744#discussion_r778655199